### ISSUE TITLE
Matrix coder duplications

### ISSUE THREAD

**Issue by mabecabrera** [SpeciesFileGroup/taxonworks#3709]
### Steps to reproduce the bug

```bash
1.When I created a matrix I used both row coder and column coder, depending on the day. 
2.When I see the final matrix I see some duplicated states. As if I had coded twice for the presence of a character. 
3. Should not this be impossible? I really do not know how this happened because I see all characters coded for each OTU, but it already happened also to Pablo Dellapé when creating a matrix. May be the state did not show complete for both coders when clicked in one coder... or maybe it is a bug created when we click many states in a short time and the program starts giving errors and says the state is already selected but it does not show. 
That happened to me a lot and also to Pablo Dellapé. The only way is to go slow.
...
```

### Screenshot

<img width="652" alt="image" src="https://github.com/SpeciesFileGroup/taxonworks/assets/63611746/db3dff64-3411-4a4a-ac04-926f92d9afc2">

### Environment

Production

### Version

v0.36.0

### Browser Used

Chrome

**Comment by mabecabrera** [comment]
I see also triplications in that screenshot

**Comment by mjy** [comment]
To confirm- you only see this in presence/absence?

**Comment by proceps** [comment]
Saad also reported this. I believe that this is the same problem we had in the past with duplication of the Authors for new source. When the internet connection is slow or interrupted, some records get duplicated. I did not experienced this with New taxonName or New source recently, but in the matrix, some scores get duplicated. To remove the score, you can go to either Edit Row or Edit Column interface and delete observation. Will you refresh browser window, you will see a next available observation for this cell. If you happen to have 3 observations, you have to delete two times...

**Comment by proceps** [comment]
It is not just presence/absence. It is for all types of descriptors.

**Comment by mabecabrera** [comment]
So we check on the matrix for this possible error and delete the observation as many times as needed and that is all?

**Comment by proceps** [comment]
Yes. I would still keep it as an issue. The bare minimum, we should probably update the interface to display multiple values if those are present in the matrix. I am not sure exactly what causes the problem, but we had a problem with duplication of the authors in the source in the past. I did not see duplication there recently. I assume that something was changed in the code. I believe it is similar issue.

**Comment by mjy** [comment]
At least with presence/absence this appears fixed?  Is anyone seeing this problem still?

**Comment by mjy** [comment]
Closing as out-of-date @mabecabrera please let use know if this is not the case.
